charts/openshift-metering/templates/presto/hive-metastore-statefulset.yaml

```diff
@@ -164,3 +172,6 @@ spec:
         persistentVolumeClaim:
           claimName: {{ .Values.presto.spec.config.sharedVolume.claimName }}
 {{- end}}
+      - name: hive-metastore-secret
+        secret:
+          secretName: metastoresecret
```

Comment:
you could store the `secretName` in the `charts/openshift-metering/values.yaml` to avoid hardcoding the name in the hive metastore statefulset

Reply:
for example, you could create a mapping in `presto.spec.hive.config` that would handle the TLS-related values. We typically use the format in other TLS sections (like reporting-operator, Presto):

```
tls:
  enabled: false
  createSecret: false
  secretName: ""
```

as the defaults values

Reply:
+1 to the condition. Same one should be used around the `ghostunnel-server` container too: https://github.com/operator-framework/operator-metering/pull/773/files#r297341285

Reply:
Also the condition is necessary for CI to pass since we don't have any secret created by default.